### DISCUSSION TITLE
Does remote logging work for scheduler/dag processor/gunicorn logs ?

### DISCUSSION THREAD

**Discussion by ginolegigot** [apache/airflow#29240] [Q&A]
Hello!
I was wondering, does remote logging also work for scheduler/dag processor/gunicorn logs ? In the GCS bucket I only dags/tasks logs, so i dont know if i did not properly configured it or if it is a normal behavior.
Thanks for your time!

**Comment by hussein-awala** [comment]
Airflow supported the remote logging for tasks to solve some problems like the need of a shared volume between the different services (especially the webserver and the different workers) to make the logs accessible from the UI, and the need to load the old log after a restart (persistent log).

But there is no need to do the same thing for the services log, where this log is used by the user for monitoring and debugging, and it's not processed by Airflow, for that the remote logging for this services is not supported. If you want to store their log on GCS, you can simply use a log collector (Logstash, Fluentd, Fluent Bit, Vector, ...) to collect the services log and upload it to GCS.

**Reply by ginolegigot** [reply]
Oki @hussein-awala, thanks for your replay, that's very clear.
Great day!
